<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountCreator</class>
 <widget class="QDialog" name="AccountCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>492</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>381</width>
    <height>481</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{background: #ffafa5; border-radius: 5px; border: 0.2px solid #d7eba9; color: #ffffff; font-size: 9pt; font-weight: bold;}																
QPushButton{background: #d9d9d9; border-radius: 5px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#81c4ff;&quot;&gt;Data&lt;/span&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#b9de6f;&quot;&gt;Bank&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{background: #F9F9F9;
border-radius: 20px;}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Серия и номер паспорта&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pasportSeries">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #d7eba9;
border-radius: 10px;
</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="pasportNumber">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #d7eba9;
border-radius: 10px;
</string>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: #81C4FF; border-radius: 15px; color: #e6f6ff; font-size: 10pt; font-weight: bold;} 
QPushButton:hover{background: #e6f6ff; border-radius: 15px; color: #81C4FF; font-size: 10pt; font-weight: bold;}</string>
    </property>
    <property name="text">
     <string>Открыть вклад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkData">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{background: #b9de6f; border-radius: 15px; border: 0.2px solid #d7eba9; color: #ffffff; font-size: 8pt; font-weight: bold;}	
																	QPushButton{background: #d9d9d9; border-radius: 15px; color: #262626; font-size: 8pt; font-weight: bold;}</string>
    </property>
    <property name="text">
     <string>Получить
информацию 
о счете</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Информация о вкладах банка&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Срок вклада&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Номер банковского счета&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deposit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Сумма&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{background: #ebc834; border-radius: 5px; border: 0.2px solid #d7eba9; color: #ffffff; font-size: 9pt; font-weight: bold;}																
QPushButton{background: #d9d9d9; border-radius: 5px; color: #262626; font-size: 9pt; font-weight: bold;}</string>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
